app/src/main/java/com/khmelenko/lab/travisclient/fragment/BranchesFragment.java

```diff
@@ -28,10 +28,10 @@
  */
 public class BranchesFragment extends Fragment implements OnListItemListener {
 
-    @Bind(R.id.branches_swipe_view)
+    @Bind(R.id.list_swipe_view)
```

Comment:
I prefer to have for this view ID as list_refreshable_swipe_view. It gives more sense that this is refreshable list and not just common implementation of the view.